diehard.js

```diff
@@ -6,7 +6,8 @@
 
   var async = require('async'),
     debug = require('debug')('diehard'),
-    Diehard;
+    Diehard,
+    timeoutHandler;
```

Comment:
can you store this internally within the object so that it is scoped to the Diehard instance? otherwise instantiating multiple Diehards could do weird things